### ISSUE TITLE
Add hotkey for 'Create Pin' on current tile

### ISSUE THREAD

**Issue by dubert** [chaorace/cqui#82]
Right now the user workflow for adding a Pin is:

1. Click the Pins button above the Minimap
2. Click add new pin button
3. Type in name
4. Choose icon
5. Press "enter" or done
6. Close Pins menu

But it could be (If user has the mouse above a tile):

1. Press hotkey (Pin menu pops up)
2. Type in name
3. Choose icon
4. Press "enter" or done

**Comment by chaorace** [comment]
Good idea, though, this will probably come after #64

**Comment by jacks0nX** [comment]
Usual behavior is that immediately upon entering the pin dialog a pin is created, should probably keep that behavior (as clunky as it is) in order to not rewrite the whole thing.

**Comment by dubert** [comment]
@jacks0nX that sounds right.

**Comment by chaorace** [comment]
@jacks0nX I guess I'll move this from low priority to future revisit, then?

**Comment by vttale** [comment]
To expand on the opening comment, the existing workflow is actually somewhat worse, because it involves the step "move to tile to pin" after "click add pin".  When you're doing a bunch of pins at once then you have to move back to "add pin" and then back to the next tile.  It's a lot of annoying extra mousing around that could really stand to be obviated.

**Comment by jacks0nX** [comment]
Well, wouldn't what you mention be fixed with our aforementioned approach?

**Comment by vttale** [comment]
Yes, I hope so, with the caveat that I'd like the hotkey to pop up not just the pin menu, but initiate the "Add Pin" function.

**Comment by jacks0nX** [comment]
Note: this does not fix the known bug that when creating a new pin an old one is actually given the chosen name/icon. I forgot about this and wondered why my implementation didn't work. Really annoying though.
